### PR TITLE
Add PreToMe and HD-Space trackers

### LINKED ISSUE #108: Indexer Request: Pretome.info
Tracker request: Pretome.info
It is available via Jackett if that helps.

Url: https://pretome.info/

### PR DESCRIPTION
**Pull request by kaso17**
Fixes #108, #121 

For a new indexer, please follow this checklist:

- [x] Run `cardigann test definitions/trackername.yml` and include the output here:

```
→ Testing 1 definition(s) (1.8.7/windows/amd64)
→ Testing indexer pretome at https://pretome.info/
  Testing required config is available ←[32mSUCCESS V←[0m ←[37min 500µs←[0m
  Testing login with valid credentials ←[32mSUCCESS V←[0m ←[37min 1.4136795s←[0m
  Testing search mode "search" ←[32mSUCCESS V←[0m ←[37min 3.1554007s←[0m
  Testing search mode "tv-search" ←[32mSUCCESS V←[0m ←[37min 3.2019065s←[0m
  Testing empty results are handled ←[32mSUCCESS V←[0m ←[37min 995.6265ms←[0m
  Testing ratio ←[32mSUCCESS V←[0m ←[37min 0s←[0m
→ Indexer pretome is ←[32mOK←[0m

→ Testing 1 definition(s) (1.8.7/windows/amd64)
→ Testing indexer hdspace at https://hd-space.org/
  Testing required config is available ←[32mSUCCESS V←[0m ←[37min 0s←[0m
  Testing login with valid credentials ←[32mSUCCESS V←[0m ←[37min 2.173776s←[0m
  Testing search mode "tv-search" ←[32mSUCCESS V←[0m ←[37min 10.3818183s←[0m
  Testing search mode "search" ←[32mSUCCESS V←[0m ←[37min 10.4783306s←[0m
  Testing empty results are handled ←[32mSUCCESS V←[0m ←[37min 2.8223584s←[0m
  Testing ratio ←[32mSUCCESS V←[0m ←[37min 4.4060595s←[0m
→ Indexer hdspace is ←[32mOK←[0m
``` 

- [X] Post the version of cardigann you tested this with (from the footer of the web interface or `cardigann --version`)
- [X] Make sure to add the indexer to the list in the README

### CODE REVIEW

File: definitions/pretome.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      password: "{{ .Config.password }}"
+      login: "Login"
+    error:
+      - selector: font[color="red"]:has(h1)
+        message:
+          selector: font[color="red"]
+    test:
+      path: /my.php
+  ratio:
+    text: "8"
```
Comment (lxt2) [inline definitions/pretome.yml]: It's always 8?? :)

Comment (kaso17) [inline definitions/pretome.yml]: An encoding issue. It's supposed to be a infinity symbol (it's a ratioless tracker).

**Comment by lxt2** [comment]
Apologies for the build failures, mind rebasing master?

**Comment by lxt2** [comment]
Never mind, I did it. Thanks for these!

**Pull request merged**